### PR TITLE
Collate changelog entries

### PR DESCRIPTION
**Pull request by drwells**
We previously discussed collating all of the small (i.e., 'fix a typo in X') changelog entries from the workshop. This supercedes #8553 by combining aforementioned changelog entries into one single entry.

I have a second PR that copies the changelog entries from #8553 into a new PR (while skipping the deleted ones and adding a few more fixes).

### CODE REVIEW

File: doc/news/changes/minor/20190810Workshop [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Improved: Make a large number of prose, grammar, and various other improvements
+to the library.
+<br>
+(Ashna Aggarwal, Mario Zepeda Aguilar, Manaswinee Bezbaruah, Katherine Cosburn,
```
Comment (tjhei) [inline doc/news/changes/minor/20190810Workshop]: I thought the format needs to be ```(name, name, name, date)``? Do our scripts handle newlines in the list of names?

**Comment by tjhei** [comment]
I went through the comments and collating them seems fine! Thanks.

File: doc/news/changes/minor/20190810Workshop (hunk 1 above)
Comment (drwells) [reply doc/news/changes/minor/20190810Workshop]: They do: I generated the doxygen and it looks fine (everything is on one line).

**Comment by drwells** [comment]
I moved the changes suggested  in #8831 here.

File: doc/news/changes/minor/20190810Workshop (hunk 1 above)
Comment (bangerth) [reply doc/news/changes/minor/20190810Workshop]: The script @tjhei mentioned is the one we use for the release checklist and the release papers. It's in a separate repository. We'll probably have to adjust a few things by hand, but I think we'll remember when we need to do that.

Comment (drwells) [reply doc/news/changes/minor/20190810Workshop]: I'll take a look.

**Pull request merged**